CONTRIBUTING.md

```diff
@@ -127,25 +128,23 @@ cocotb aims to keep the `master` branch always in a releasable state.
 At least four times a year an official release should be created.
 It is the job of the maintainers to find a suitable time for a release, to communicate it to the community, and to coordinate it.
 
-
 Maintainers
 -----------
 
 Cocotb uses a shared maintainer model.
 Most maintainers are experts in part of the cocotb codebase, and are primarily responsible for reviews in this area.
 
 - Kaleb Barrett (@ktbarrett)
-- Julius Baxter (@juliusbaxter)
-- Luke Darnell (@lukedarnell)
 - Tomasz Hemperek (@themperek)
-- Chris Higgs (@chiggs).
-  Founder of cocotb.
-- Stuart Hodgson (@stuarthodgson).
-  Founder of cocotb.
 - Colin Marquardt (@cmarqu)
 - Philipp Wagner (@imphil)
 - Eric Wieser (@eric-wieser)
 
+Founders
+
+- Chris Higgs (@chiggs)
+- Stuart Hodgson (@stuarthodgson)
```

Comment:
Could consider:

```suggestion
Founders:

- Chris Higgs (@chiggs)
- Stuart Hodgson (@stuarthodgson)

Emeritus maintainers:

- Julius Baxter (@juliusbaxter)	
- Luke Darnell (@lukedarnell)
```

Reply:
Given that Jules and Luke have never been "maintainer" in the sense that we have assigned to the term since this document was written, and the discussion I had with them, I'll keep the removal for now.